content/posts/GSoC_Coding_Phase_Blog_4/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "GSoC Coding Phase 2 Blog 2"
+date: 2020-07-23T19:47:51+05:30
+draft: false
+categories: ["News", "GSoC"]
+description: "Progress Report for the second half of the Google Summer of Code 2020 Phase 2 for the Baseline Images Problem"
+displayInList: true
+author: Sidharth Bansal
+---
+
+Google Summer of Code 2020's second evaluation is about to complete. Now we are about to start with the final coding phase. This post discusses about the progress so far in the last two weeks of the second coding period from 13 July to 26 July 2020.
+
+## Modular approach towards removal of matplotlib baseline images
+
+We have divide the work in two parts as discussed in the previous blog. The first part is the generation of the baseline images discussed below. The second part is the modification of the baseline images. Modification part will be implemented last phase of the Google Summer of Code 2020.
```

Comment:
link to previous post?  Is there a nice way to do this with in the blogging framework?

Reply:
You could add a link to the previous blog using the URL. Also by adding the tag GSOC it will be grouped together with the others here https://matplotlib.org/matplotblog/categories/gsoc/